content/source/docs/cloud/vcs/azure-devops-server.html.md

```diff
@@ -44,15 +44,15 @@ Terraform Cloud uses personal access tokens to connect to Azure DevOps Server. T
 
     ![Azure DevOps Server Screenshot: Adding a new VCS Provider in Terraform Cloud](./images/azure-devops-server-add-vcs-provider.png)
 
-2. The next page has a drop-down and several text fields. Select "Azure DevOps Server" from the drop-down.
+2. The next page has a drop-down and several text fields. Select "**Azure DevOps Server**" from the drop-down.
```

Comment:
Is having quotes and bold too much? I would think one or the other, or are you following formatting from other docs?

Reply:
yeah that's fair, in fact when comparing it to Services it was exactly how Patrick had it. I'll put it back.